### PR TITLE
cloud: update doc for multi org

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

as title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### CODE REVIEW

File: tidb-cloud/manage-user-access.md [hunk 1]
```diff
@@ -69,10 +69,25 @@ Under this structure:
...
+
+## Switch organizations
+
+If you are a member of multiple organizations, you can switch your account between organizations.
+
+To switch organizations, perform these steps:
+
+1. Click the account name in the upper-right corner of the TiDB Cloud console.
+2. Hold the mouse over **Switch Organization**, and you can see all the organizations you belong to in the submenu.
+3. Click the organization you want to switch to.
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:90]:
```suggestion
2. Hold the mouse over **Switch Organization**, and click the organization you want to switch to.
```

**Review by langyuezhang — APPROVED** [review]
LGTM

File: tidb-cloud/manage-user-access.md [hunk 2]
```diff
@@ -69,11 +69,25 @@ Under this structure:
 
 To access a cluster in a project under an organization, a user must be both a member of the organization and a member of the project. Organization owners can invite users to join the project to create and manage clusters in the project.
 
+To view which organizations you belong to, perform these steps:
+
+1. Click the account name in the upper-right corner of the TiDB Cloud console.
+2. Hold the mouse over **Switch Organization**, and you can see all the organizations you belong to in the submenu.
```
Comment (TomShawn) [inline tidb-cloud/manage-user-access.md:75]: Some people don't use mouses.
```suggestion
2. Move your cursor to **Switch Organization**, and you can see all the organizations you belong to in the submenu.
```

File: tidb-cloud/manage-user-access.md [hunk 3]
```diff
@@ -69,11 +69,25 @@ Under this structure:
...
 2. Click **Organization Settings**. The **Projects** tab is displayed by default.
 
+## Switch organizations
+
+If you are a member of multiple organizations, you can switch your account between organizations.
+
+To switch organizations, perform these steps:
+
+1. Click the account name in the upper-right corner of the TiDB Cloud console.
+2. Hold the mouse over **Switch Organization**, and click the organization you want to switch to.
```
Comment (TomShawn) [inline tidb-cloud/manage-user-access.md:89]:
```suggestion
2. Move your cursor to **Switch Organization**, and click the organization you want to switch to.
```

File: tidb-cloud/manage-user-access.md [hunk 4]
```diff
@@ -98,7 +112,7 @@ To invite a member to an organization, perform the following steps:
 
 7. After receiving this email, the user needs to click the link in the email to verify the identity, and a new page shows.
 
-8. On the new page, the user needs to view and agree with our license, and then click **Submit** to create the account in TiDB Cloud. After that, the user is redirected to the login page.
+8. If the invited email address does not have a TiDB Cloud account, the user is directed to a sign up page to create an account. If the email address already has a TiDB Cloud account, the user is directed to the sign in page, and after sign-in, the account joins the organization automatically.
```
Comment (TomShawn) [inline tidb-cloud/manage-user-access.md:115]:
```suggestion
8. If the invited email address has not been signed up for a TiDB Cloud account, the user is directed to the sign-up page to create an account. If the email address has been signed up for a TiDB Cloud account, the user is directed to the sign-in page, and after sign-in, the account joins the organization automatically.
```

**Review by TomShawn — APPROVED** [review]
minor comments

File: tidb-cloud/manage-user-access.md [hunk 5]
```diff
@@ -69,11 +69,25 @@ Under this structure:
...
+
+1. Click the account name in the upper-right corner of the TiDB Cloud console.
+2. Select **Switch Organization** in the drop-down menu, and you can see all the organizations you belong to in the submenu.
+
 To check which project you belong to, perform these steps:
 
 1. Click the account name in the upper-right corner of the TiDB Cloud console.
 2. Click **Organization Settings**. The **Projects** tab is displayed by default.
 
+## Switch organizations
```
Comment (TomShawn) [inline tidb-cloud/manage-user-access.md:82]:
```suggestion
## Switch between organizations
```

File: tidb-cloud/manage-user-access.md [hunk 6]
```diff
@@ -69,11 +69,25 @@ Under this structure:
...
+
+If you are a member of multiple organizations, you can switch your account between organizations.
+
+To switch organizations, perform these steps:
```
Comment (TomShawn) [inline tidb-cloud/manage-user-access.md:86]:
```suggestion
To switch between organizations, perform these steps:
```

**Pull request merged**